components/module.py

```diff
@@ -0,0 +1,27 @@
+class Module:
+    # instances
+    # module_number = 0
+
+    # **kwargs for json
+    def __init__(self, module_number, **kwargs):
+        self.module_number = module_number
+        module_number += 1
```

Comment:
this also isn't doing anything, you're incrementing a local variable that's destroyed at the end of the constructor